commands/backupunit.go

```diff
@@ -193,7 +193,10 @@ func PreRunBackupUnitNameEmailPwd(c *core.PreCommandConfig) error {
 }
 
 func RunBackupUnitList(c *core.CommandConfig) error {
-	backupUnits, _, err := c.BackupUnit().List()
+	backupUnits, resp, err := c.BackupUnit().List()
+	if resp != nil {
+		c.Printer.Verbose("the execution time of the command is: %v", resp.RequestTime)
```

Comment:
In my opinion, this should be updated to something like: "The execution time of the request is"

Reply:
done